android/build.gradle

```diff
@@ -1,15 +1,15 @@
 group = "io.piano.flutter.piano_analytics"
-version = "1.0.1"
+version = "1.0.2"
 
 buildscript {
-    ext.kotlin_version = "1.9.0"
+    ext.kotlin_version = "2.1.0"
```

Comment:
It's better to use 2.0.21, as android library [does](https://github.com/at-internet/piano-analytics-android/blob/main/gradle/libs.versions.toml#L3)